CONTRIBUTING.md

```diff
@@ -25,15 +25,14 @@ There are two ways in which you can help update the content:
 If you are not a regular contributor to the project, but you would like to
 contribute some changes, the best way to do it is:
 
-  1. Fork the repo or create a new feature branch
-  2. Write in your contributions
-  3. Create a PR into this repo
+  1. Fork this repository or create a new feature branch
+  2. Add your contributions
+  3. Submit a PR to merge your contribution to this repository
 
 * **Periodic and regular contributions:** \
 If you plan to update the content semi-regularly or regularly, you can be added
-to the project's Contributors group. Please contact one of the project's
-administrators to be added to the group.
-You will still need a PR against master in order to merge your changes.
+to the project's Contributors group. Please contact [Federica Nocera](https://github.com/fnocera) or Tim Pearson to be added to the group.
```

Comment:
Brock Davis @brockneedscoffee is also a maintainer. I will reach out to Beat to get him admin rights so that we can both be listed as contacts.

Reply:
Sounds good, @fnocera I could also be a maintainer if you want